rtsf-at-checkout-device-scale/Dockerfile

```
@@ -19,18 +19,18 @@ RUN go mod download
 COPY . .
 
 RUN go mod tidy
-RUN go build -tags no_zmq -o device-scale cmd/main.go
+RUN CGO_ENABLED=0 go build -tags no_zmq -o device-scale cmd/main.go
```

Tag no longer exists now that ZMQ has been totally removed
```suggestion
RUN CGO_ENABLED=0 go build -o device-scale cmd/main.go
```